src/app/Library/CrudPanel/Traits/Validation.php

```diff
@@ -133,7 +133,7 @@ public function setValidation($classOrRulesArray = false, $messages = [])
             $this->setValidationFromFields();
         } elseif (is_array($classOrRulesArray)) {
             $this->setValidationFromArray($classOrRulesArray, $messages);
-        } elseif (is_string($classOrRulesArray) || is_class($classOrRulesArray)) {
+        } elseif (is_string($classOrRulesArray) || class_exists($classOrRulesArray)) {
```

Comment:
```suggestion
        } elseif (is_string($classOrRulesArray) && class_exists($classOrRulesArray)) {
```

This should probably be an `AND` instead of `OR` right ?